modules/cli/cmd/sequencer/set-config.go

```diff
@@ -0,0 +1,118 @@
+package sequencer
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/astriaorg/astria-cli-go/modules/cli/cmd"
+	"github.com/astriaorg/astria-cli-go/modules/cli/cmd/devrunner/config"
+	"github.com/pelletier/go-toml/v2"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// setConfigCmd represents the set-config root command
+var setConfigCmd = &cobra.Command{
+	Use:   "set-config",
+	Short: "Update the configuration for the sequencer commands config.",
+}
+
+// setAssetCmd represents the set-config asset command
+var setAssetCmd = &cobra.Command{
+	Use:   "asset [denom]",
+	Short: "Sets the asset and fee asset in the sequencer command configs.",
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+	Run:   setAssetCmdHandler,
+}
+
+func setAssetCmdHandler(c *cobra.Command, args []string) {
+	flagHandler := cmd.CreateCliFlagHandler(c, cmd.EnvPrefix)
+
+	sequencerNetworksConfigPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", DefaultSequencerNetworksConfigFilename)
+	// create the networks config file if it doesn't exist. Will skip if the
+	// file already exists.
+	CreateSequencerNetworkConfigs(sequencerNetworksConfigPath)
+	sequencerNetworkConfigs := LoadSequencerNetworkConfigsOrPanic(sequencerNetworksConfigPath)
+
+	network := flagHandler.GetValue("network")
+	config.IsInstanceNameValidOrPanic(network)
+
+	asset := args[0]
+	config.IsInstanceNameValidOrPanic(asset)
+
+	log.Info("Updating sequencer_chain_id to", asset, "in sequencer-networks-config.toml for network: ", network)
+	config := sequencerNetworkConfigs.Configs[network]
+	config.Asset = asset
+	config.FeeAsset = asset
+	sequencerNetworkConfigs.Configs[network] = config
+
+	file, err := os.Create(sequencerNetworksConfigPath)
+	if err != nil {
+		log.Error("Error creating file:", sequencerNetworksConfigPath, ":", err)
```

Comment:
You can combine this line and the next line I think with something like `log.Fatal` or log.Panic`